I 38 watch namespace (#39)
tuenti/secrets-manager: README.md

| `vault.token-polling-period` | 15s | Polling interval to check token expiration time. |
| `vault.renew-ttl-increment` | 600 | TTL time for renewed token. |
| `metrics-addr` | `:8080` | The address to listen on for HTTP requests. |

> | `watch-namespaces` | `""` | Comma separated list of namespaces where secrets-manager is allowed to watch. By default all namespaces will be watched. |

Reviewer: not sure about the use of the word `allowed` here (in the exclude flag), since it implies some form of authorization type of restriction, perhaps something like:
```suggestion
| `watch-namespaces` | `""` | Comma separated list of namespaces which secrets-manager will watch for Secret Definitions. By default all namespaces will be watched. |
```

wdyt?

Author: makes sense thanks!!

> | `exclude-namespaces` | `""` | Comma separated list of namespaces where secrets-manager is not allowed to watch. By default all namespaces will be watched. |

Reviewer: to align with above suggestion:
```suggestion
| `exclude-namespaces` | `""` | Comma separated list of namespaces which secrets-manager will not watch for Secrets Definitions. By default all namespaces will be watched. |
```

> | `exclude-namespaces` | `""` | Comma separated list of namespaces that secrets-manager will not watch for `SecretDefinitions`. By default all namespaces are watched. |

Author: I think it's a good point! :+1:

> | `exclude-namespaces` | `""` | Comma separated list of namespaces that secrets-manager will not watch for `SecretDefinitions`. By default all namespaces are watched. |

Reviewer 2: i'll do it in the next PR
